crates/uv-python/src/managed.rs

```diff
@@ -475,28 +479,62 @@ impl ManagedPythonInstallation {
         Ok(())
     }
 
-    /// Create a link to the Python executable in the given `bin` directory.
-    pub fn create_bin_link(&self, bin: &Path) -> Result<PathBuf, Error> {
+    /// Create a link to the managed Python executable.
+    pub fn create_bin_link(&self, target: &Path) -> Result<(), Error> {
         let python = self.executable();
 
+        let bin = target.parent().ok_or(Error::NoExecutableDirectory)?;
         fs_err::create_dir_all(bin).map_err(|err| Error::ExecutableDirectory {
             to: bin.to_path_buf(),
             err,
         })?;
 
-        // TODO(zanieb): Add support for a "default" which
-        let python_in_bin = bin.join(self.key.versioned_executable_name());
+        if cfg!(unix) {
+            match uv_fs::symlink_copy_fallback_file(&python, target) {
```

Comment:
Does this still work when we copy, wouldn't the python interpreter be unable to find its home, esp with pbs?

Reply:
This actually will only copy on Windows, I think. I'll update the utility accordingly or not use it here for clarity.